pkg/lwip/include/lwipopts.h

```diff
@@ -137,6 +137,9 @@ extern "C" {
 #define LWIP_NETCONN            0
 #endif /* MODULE_LWIP_SOCK */
 
+#ifdef MODULE_SHELL_COMMANDS
+#define LWIP_DEBUG              1
```

Comment:
Does this mean there will also be debug output be printed, or does this only allow you to use `ip_addr_debug_print()`? In case of the first, I would prefer a different solution.

Reply:
No, this flag just changes lwip debug functions from being no-ops to actually evaluating the debug level. LWIP_DBG_ON is used to enable printing the addresses, all debug flags (like NETIF_DEBUG) are off by default.